content/docs/examples/multicluster/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Enabling multiclusters
-description: A variety of fully working multicluster examples for Istio that you can experiment with.
+title: Spanning Istio mesh across multiple clusters in same network
+description: A variety of fully working VPN/flat-network based multicluster examples for Istio that you can experiment with.
```

Comment:
Why are you hijacking this example instead of creating a separate example.  The setup pages have both options and the example already have multiple cloud examples.   Please create this as an additional example not as a replacement.

Reply:
can you please read the entire PR before coming to these conclusions? I don't have any example for the zero VPN case. Its simply setup. The rest is fixing up links, titles, etc.

Reply:
Apologize - I had a rendering problem when looking at this locally.  Please ignore this comment.

Reply:
Missing article:
Spanning an Istio mesh across multiple clusters in same network
Consider a different word instead of "Spanning". I suggest Installing or Configuring
Spanning is an abstract verb, at least the way you are using it, and can be confusing for non-English native speakers.